src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -277,9 +277,16 @@ bool Replayer<I>::get_replay_status(std::string* description,
       matching_remote_snap_it !=
         m_state_builder->remote_image_ctx->snap_info.end()) {
     root_obj["syncing_snapshot_timestamp"] = remote_snap_info->timestamp.sec();
-    root_obj["syncing_percent"] = static_cast<uint64_t>(
-        100 * m_local_mirror_snap_ns.last_copied_object_number /
-        static_cast<float>(std::max<uint64_t>(1U, m_local_object_count)));
+
+    if (m_local_object_count > 0) {
+      root_obj["syncing_percent"] = static_cast<uint64_t>(
+	100 * m_local_mirror_snap_ns.last_copied_object_number /
+	static_cast<float>(m_local_object_count));
```

Comment:
What is the reason for going through float?  Can this be simplified to:
```suggestion
      root_obj["syncing_percent"] =
          100 * m_local_mirror_snap_ns.last_copied_object_number /
          m_local_object_count;
```

Reply:
The original code had the static_cast so I did not change that.

Reply:
The original code also displayed a bogus value...  I don't see a reason to cling onto the construct whose purpose appears to be unclear.